scripts/track.json

```diff
@@ -35,7 +33,7 @@
   },
   "event": "Demo Track - New",
   "integrations": {
-    "All": true,
+    "All": false,
```

Comment:
Let's remove this? It will confuse someone if they use this and the event is not sent to destination.

Reply:
done